docs/release_notes/2024/nov2024.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 1
+id: November_2024
+description: Release notes for November
+title: November 2024
+tags:
+  - release notes
+  - changelog
+  - november
+---
+
+## 3.4.1.\* (November 25, 2024)
+
+- Prophecy Python libs version: 1.9.24
+- Prophecy Scala libs version: 8.4.0
+
+### Features {#Features341}
+
+#### Monitor and alert for resource utilization
+
+Prophecy now offers a reliable solution for configuring alerts to monitor resource usage in Prophecy-managed microservices.
+
+![Alert Config](./img/nov-alert-config.png)
+
+It enables proactive alerting when resource utilization approaches defined thresholds, ensuring timely intervention before limits are reached. Additionally, it supports the suspension of critical services in the event of resource overflows.
+
+For more information, see Alerts Configuration.
+
+#### Download Logs
+
+As an admin user, you can download your environment logs from Prophecy without needing access to your Prophecy cluster or the assistance of Prophecy Support.
+
+![Download Logs](./img/nov-download-logs.png)
+
+This reduces the delay in debugging any issues with your Prophecy services. You can share the compressed log file with your Prophecy support engineer by uploading it to Zendesk.
+
+For more information, see Download Logs.
+
+#### Extended Maintenance release
+
+Release 3.4.1 is our first Extended Maintenance (EM) release. These releases provide you with a long-term support Prophecy version.
```

Comment:
Will all future releases be EM releases? Or is it only some releases? Or, is it all releases on an opt-in basis? 

Or maybe I do not understand at all 🫠

Reply:
Done.